client/src/actions/activityActions.js

```diff
@@ -0,0 +1,13 @@
+import { GET_ACTIVITY } from "./types";
+import axios from "axios";
+
+export const getActivity = (id) => dispatch => {
+  console.log("inside activity");
```

Comment:
The console.log entries should have meaningful debugging information so that the console doesn't end up getting cluttered with information overload.  Best practice is generally not to commit them though.